api/src/main/java/io/druid/segment/loading/DataSegmentPusherUtil.java

```diff
@@ -54,14 +54,25 @@ public static String getStorageDir(DataSegment segment)
   public static String getHdfsStorageDir(DataSegment segment)
```

Comment:
`getHdfsStorageDir` seems unused now, why not replace it with the code from `getHdfsStorageDirUptoVersion`?